linetools/spectralline.py

```diff
@@ -71,7 +72,7 @@ def from_dict(cls, idict):
         """
         # Init
         if idict['ltype'] == 'Abs':
-            sline = AbsLine(idict['trans'])
+            sline = AbsLine(idict['name'])
```

Comment:
This may fail for a number of JSON files that I already have 
in use.  Can you make this a try for now together with an
except on 'trans'?